models/importer.js

```diff
@@ -0,0 +1,42 @@
+import Promise from 'bluebird';
+import fs from 'fs';
+import Log from 'log';
+import config from '../config/config.json';
+
+const log = new Log();
+const readFileAsync = Promise.promisify(fs.readFile);
+
+export class Importer {
+  static import(path) {
+    return readFileAsync(path, 'utf8')
+      .then(content => Importer.csvToJson(content));
+  }
+
+  static importSync(path) {
+    if (fs.existsSync(path)) {
+      const content = fs.readFileSync(path, 'utf8');
+      return Importer.csvToJson(content);
+    }
+    log.error(config.file_not_found);
+    return '';
+  }
+
+  static csvToJson(content) {
+    const result = [];
+    const lines = content.split('\n');
+    const headers = lines[0].split(',');
+
+    for (let i = 1; i < lines.length; i += 1) {
```

Comment:
Maybe it's a personal preference but I think that in fuctional style it's easier to read
```js
const result = lines
  .slice(1)
  .map(line => line.split(',').reduce((obj, value, index) => {
    obj[headers[index]] = value;
    return obj;
  }, {}))
```
With lodash/fp it'd be even better.
```js
flow(
  tail,
  map(line => zipObject(headers, line.split(',')))
)(lines)
```

Reply:
Makes sense. Thanks for the advice